objects/src/accounts/data.rs

```diff
@@ -136,32 +135,15 @@ mod tests {
     use super::{AccountData, AuthData};
     use crate::{
         accounts::{
-            storage, Account, AccountCode, AccountId, Felt, Word,
-            ACCOUNT_ID_REGULAR_ACCOUNT_IMMUTABLE_CODE_ON_CHAIN,
+            account_id::testing::ACCOUNT_ID_REGULAR_ACCOUNT_IMMUTABLE_CODE_ON_CHAIN,
+            code::testing::make_account_code, storage, Account, AccountId, Felt, Word,
         },
         assets::AssetVault,
     };
 
     fn build_account_data() -> AccountData {
-        // create account id
         let id = AccountId::try_from(ACCOUNT_ID_REGULAR_ACCOUNT_IMMUTABLE_CODE_ON_CHAIN).unwrap();
-
-        // build account code
-        let source = "
-            export.foo
-                push.1 push.2 mul
-            end
-
-            export.bar
-                push.1 push.2 add
-            end
-        ";
```

Comment:
This was the closest I got to make the tests more "generic". As in, I moved the duplicated code to a single `make_account_code` function that is used on the tests that needed it.